R/ui.R

```diff
@@ -134,23 +134,20 @@ unmount_openapi <- function(pr) {
 #' When you load these packages, it calls [register_ui()] to provide a user
 #' interface that can interpret your plumber OpenAPI Specifications.
 #'
-#' `ui` list expects the following values
-#' \describe{
-#' \item{name}{Name of the UI.}
-#' \item{index}{A function that returns the HTML content of the landing page of the UI.}
-#' \item{static}{A function that returns the path to the static assets (images, javascript, css, fonts) the UI will use.}
-#' }
+#' @param name Name of the UI
+#' @param index A function that returns the HTML content of the landing page of the UI.
+#' @param static A function that returns the path to the static assets (images, javascript, css, fonts) the UI will use.
+#'
```

Comment:
Do we have any documentation for the input arguments to these functions?

Reply:
Documentation should be in each UI package, maybe?

https://github.com/meztez/rapidoc/blob/d370734dcfb4772ffa98d7f165c40821cc10136a/R/paths.R#L40

Or are you talking about how they are called from inside plumber?

https://github.com/rstudio/plumber/blob/13f813400062c9a287af4602172a4232b4c6a545/R/ui.R#L161

It will feed of whatever you put in set_ui as extra arguments.

Reply:
This whole thing probably requires a vignette in plumberDeploy

Reply:
> Or are you talking about how they are called from inside plumber?

This one. Have added an example and more docs.

-------------------------


@meztez If a vignette is made, I think it should stay in `plumber` as the content is for API UIs, not deployment of plumber APIs.